src/main/scala/nl/knaw/dans/easy/stage/lib/JSON.scala

```diff
@@ -99,7 +96,7 @@ object JSON {
 
     def redirectFileDatastreamJson = {
       createJSON(
-        ("dsLocation" -> dsLocation.toURI.toASCIIString) ~
+        ("dsLocation" -> settings.datastreamLocation.get.toURI.toASCIIString) ~
```

Comment:
One of `settings.datastreamLocation` and `settings.file` should always be filled. Maybe there is a more elegant way to program this. I am open to suggestions.